packages/code-du-travail-api/src/server/routes/search/index.js

```diff
@@ -0,0 +1,33 @@
+const Router = require("koa-router");
+const API_BASE_URL = require("../api").BASE_URL;
+const routeName = "/search";
+const router = new Router({ prefix: API_BASE_URL });
+
+const elasticsearchClient = require("../../conf/elasticsearch.js");
+const getSearchBody = require("./searchBody");
+
+const index =
+  process.env.ELASTICSEARCH_DOCUMENT_INDEX || "code_du_travail_numerique";
+
+/**
+ * Return documents matching the given query.
+ *
+ * @example
+ * http://localhost:1337/api/v1/search?q=incapacité%20travail
+ *
+ * @param {string} querystring.q A `q` querystring param containing the query to process.
+ * @returns {Object} Results.
+ */
+router.get(routeName, async ctx => {
+  let query = ctx.request.query.q;
+  const size = Math.min(ctx.request.query.size || 10, 100);
+
+  let excludeSources = (ctx.request.query.excludeSources || "").split(",");
```

Comment:
I though there is a linter rule that handle that kin of nitpick

Reply:
I would gladly add a "no let" rule.
I think it would be better then me always commenting on every "let" I see :stuck_out_tongue: 
Still it might lift a lot of them to "fix" :smirk:

Reply:
we can do that on API, there no other error about `let`/ `const`